client/components/editable-service-name/editable-service-name.less

```diff
@@ -14,15 +14,14 @@
 
         .btn-editable {
             padding: 0;
```

Comment:
custom rule ??? :(

Defines a line-height to align middle :p

Reply:
Yes it is a custom rule that we previously used. We know that we have to remove it and prefer doing this with a component to achieve this. We try to implement this on phase 2.